new-pipeline/src/dist.js

```diff
@@ -869,15 +869,22 @@ function displayHistograms(histogramsList, dates, useTable, cumulative, trim) {
     var channel = $("#channel-version").val().split("/")[0]
     var desc = getDescription(metric, channel, description);
     var link = getDescriptionLink(metric, channel, description);
-    if (metric != desc) {
-      $('#dist-caption-text').html(desc);
-    } else {
-      $('#dist-caption-text').text("");
-    }
+    var useCounterLink = getUseCounterLink(metric, channel, description);
+    $('#dist-caption-text').html(desc);
     $('#dist-caption-link').html(link);
+    $('#use-counter-link').html(useCounterLink);
+
   } else {
     $('#dist-caption-text').text(""); // Clear the histogram caption
     $('#dist-caption-link').text(""); // Clear the histogram link
+    $('#use-counter-link').text(""); // Clear use counter link
+  }
+
+  if (!useCounterLink) {
```

Comment:
I think we don't need the code here.
This should work with just the changes above.